schedule/yast/sle/guided_ext4/guided_ext4_s390x_zvm.yaml

```diff
@@ -1,9 +1,8 @@
 ---
-name:           guided_ext4
-description:    >
+name: guided_ext4
```

Comment:
this file is missing to be unified.

Reply:
The problem with this file is that it uses `validate_partition_table_via_parted` instead of `validate_partition_table_via_blkid`, which blocks the unification. Of course I could edit the test data like we did in `guided_btrfs` for s390x-ZVM, but that would mean that we don't validate anything at all since in `YAML_TEST_DATA` there are no "disks:" defined, so nothing to checke and nothing that can fail. But in that case I could also avoid running `validate_partition_table_via_blkid` at all.

Reply:
and the new file create recently (I believe I mentioned to you...) ? `tests/console/validate_partition_table.pm`
would this help? if not I need to read more deeply your comment.

Reply:
Tried with that test module and it worked. So I updated the PR.